src/react/components/common/assetPreview/tfrecordAsset.tsx

```diff
@@ -0,0 +1,81 @@
+import React from "react";
+import { IAssetProps } from "./assetPreview";
+import { IAsset, AssetType } from "../../../../models/applicationState";
+import HtmlFileReader from "../../../../common/htmlFileReader";
+import { TFRecordsReader } from "../../../../providers/export/tensorFlowRecords/tensorFlowReader";
+import { FeatureType } from "../../../../providers/export/tensorFlowRecords/tensorFlowBuilder";
+
+/**
+ * Properties for TFRecord Asset Image component
+ */
+export interface ITFRecordProps extends IAssetProps, React.ClassAttributes<TFRecordAsset> {
```

Comment:
Is there a reason to use this empty props as opposed to `IAssetProps`?

Reply:
Not really, you're right it's empty interface.  I replicated same behavior as in imageAsset where it's there too even if empty but you're right.  Let me just remove this on both on the other PR.  Thanks